### DISCUSSION TITLE
[Android] automatically configure Debug keystore if it's not configured

### DISCUSSION THREAD

**Discussion by poyribn** [godotengine/godot-proposals#7052] [Platforms]
suggested by @Calinou in https://github.com/godotengine/godot/issues/76365#issuecomment-1585000108:

> A debug keystore can be generated by a single command, so I don't see much point in committing it to version control. In fact, it should probably be the Godot editor's job to automatically create this debug keystore and reference it in the Editor Settings when you try to export a project (if no debug keystore is currently configured).

**Comment by m4gr3d** [comment]
Fixed by https://github.com/godotengine/godot/pull/90611
